src/main/scala/hyperledger/identus/vdr/prism/PrismDriver.scala

```diff
@@ -0,0 +1,255 @@
+            (updateEventHash, code, string) = ret
+            out = Driver.OperationResult(
+              updateEventHash.hex,
+              Driver.OperationState.SUCCESS,
+              Array(eventRef.eventHash.hex),
+              Map(("h", eventRef.eventHash.hex)).asJava,
+              null,
+              null,
+              null
+            )
+          } yield out
+        )
+  }
+
+  override def read(
+      paths: Array[String],
+      queries: java.util.Map[String, String],
+      fragment: String,
+      publicKeys: Array[java.security.PublicKey]
+  ): Array[Byte] = {
+    paths.headOption match
+      case None => Array.empty()
+      case Some(hash) =>
+        val eventRef: RefVDR = RefVDR(hash)
+        GenericVDRDriver.runProgram(
+          for {
+            vdr <- genericVDRDriver.read(eventRef)
+          } yield vdr.data match {
+            case VDR.DataEmpty()              => Array.empty()
+            case VDR.DataDeactivated(data)    => Array.empty()
+            case VDR.DataByteArray(byteArray) => byteArray
+            case VDR.DataIPFS(cid)            => Array.empty()
+            case VDR.DataStatusList(status)   => Array.empty()
+          }
+        )
+  }
+
+  /** in the case of PRISM this identifier is the HASH of the event */
+  override def storeResultState(
+      identifier: String
+  ): interfaces.Driver.OperationState =
+    OperationState.SUCCESS
+    // OperationState.RUNNING //TODO since everything is a synchronous, the driver needs to keep a internal state for this
+    // OperationState.ERROR //TODO only makes sense when the event submitted is not the latest version or if the submission didn't end up the the blockchain
+
+  override def verify(
+      paths: Array[String],
+      queries: java.util.Map[String, String],
+      fragment: String,
+      publicKeys: Array[java.security.PublicKey],
+      returnData: Boolean
+  ): interfaces.Proof = {
+    paths.headOption match
+      case None => ???
+      case Some(hash) =>
+        val eventRef: RefVDR = RefVDR(hash)
+        GenericVDRDriver.runProgram(
+          for {
+            vdr <- genericVDRDriver.read(eventRef)
+          } yield vdr.data match {
+            case VDR.DataEmpty() => Proof("PrismBlock", Array.empty(), Array.empty()) // TODO proof
+            case VDR.DataDeactivated(data) =>
+              data match {
+                case VDR.DataEmpty()           => throw DataNotInitializedException()
+                case VDR.DataDeactivated(data) => throw DataAlreadyDeactivatedException()
+                case VDR.DataByteArray(byteArray) =>
+                  Proof(
+                    "PrismBlock",
+                    byteArray, // Data
+                    Array.empty() // TODO proof will is a protobuf Array of PRISM events. Reuse the PrismBlock?
+                  )
+                case VDR.DataIPFS(cid)          => ??? // not part of the generic VDR specification
```

Comment:
In the future, we could try to implement, other types of Data we want to support.
But since we can only return bytes, we need to figure out the best way to create the URL. that hints to the user this will return a CID for IPF, or any other type of data.